### PR TITLE
[#303]fix-Failed_to_get_the_dns_service_IP_list

### PR DESCRIPTION
**Pull request by carryaimp**
1. Added the logic for determining whether the obtained DNS IP address list is empty
2. Get and set the label of the default selected DNS service through the environment variable

### CODE REVIEW

**Comment by ii2day** [comment]
@carryaimp  我们有专门获取 env 环境变量的方法，参考这个文件对 env 进行设置，https://github.com/kdoctor-io/kdoctor/blob/main/pkg/types/agent_config.go

**Comment by ii2day** [comment]
@carryaimp We have a special method to obtain env environment variables. Refer to this file to set env.https://github.com/kdoctor-io/kdoctor/blob/main/pkg/types/agent_config.go

**Comment by carryaimp** [comment]
@ii2day It has been modified according to the specifications

File: pkg/types/agent_config.go [hunk 1]
```diff
@@ -26,6 +26,8 @@ var AgentEnvMapping = []EnvMapping{
 	{"ENV_CLUSTER_DNS_DOMAIN", "cluster.local", &AgentConfig.ClusterDnsDomain},
 	{"ENV_LOCAL_NODE_IP", "", &AgentConfig.LocalNodeIP},
 	{"ENV_LOCAL_NODE_NAME", "", &AgentConfig.LocalNodeName},
+	{"DNS_SERVICE_SELECT_LABEL_KEY", "kubernetes.io/name", &AgentConfig.DnsServiceSelectLabelKey},
```
Comment (ii2day) [inline pkg/types/agent_config.go:29]: Unified format, use ENV as the beginning

Comment (carryaimp) [reply pkg/types/agent_config.go]: OK, environment variable prefixes have been unified

**Review by weizhoublue — APPROVED** [review]

**Pull request merged**